src/app/_blocks/HubContentGrid/styles.module.css

```diff
@@ -0,0 +1,51 @@
+/* Content grid */
+.contentGridContainer {
+  position: relative;
+  background-color: #FFFFFF;
+  width: max(calc(100% - 32px), calc(100% - 160px));
+  margin: 40px auto;
+  border-bottom-right-radius: 45px;
+  border-bottom-left-radius: 45px;
+  border-top-right-radius: 45px;
+  border: 1px solid var(--dark-rock-800) !important;
```

Comment:
Do we really need the !important? What is it overriding?

Reply:
It's the only way i've found to override the border color change that happens on the desktop (changing conditionally based on `activeButton`) , otherwise the borders would also change on mobile 

![imagem](https://github.com/user-attachments/assets/0713a778-b53c-4c3b-87ad-648f2a4b9d0e)